tenzir/tests/exec/udo/bad/udo.txt

```diff
@@ -0,0 +1,9 @@
+
+[1m[31merror[39m: operator `bar` not found[0m
+ [1m[34m=[39m hint:[0m did you mean `rare`?
+ [1m[34m=[39m docs:[0m https://docs.tenzir.com/tql2/operators
+
+[1m[31merror[39m: operator `foo` not found[0m
+ [1m[34m=[39m hint:[0m did you mean `fork`?
+ [1m[34m=[39m docs:[0m https://docs.tenzir.com/tql2/operators
+failed to resolve user-defined operators: `foo`, `bar`
```

Comment:
Why do you get escape codes here, and why is there a blank line on top?

Reply:
I think this is just badly written code in the main function where we print this. Want me to fix that as a drive-by?

Reply:
Ah, that might have been me passing in `color_diagnostics::yes` unconditionally somewhere… and probably leaving a TODO for it… well…